{"name":"getEmployeeDetails","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [success] but found [fail]","trace":"java.lang.AssertionError: expected [success] but found [fail]\r\n\tat org.testng.Assert.fail(Assert.java:97)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:136)\r\n\tat org.testng.Assert.assertEquals(Assert.java:118)\r\n\tat org.testng.Assert.assertEquals(Assert.java:575)\r\n\tat org.testng.Assert.assertEquals(Assert.java:585)\r\n\tat com.enquero.api.driscolls.tests.EmployeeTest.getEmployeeDetails(EmployeeTest.java:79)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.lang.reflect.Method.invoke(Method.java:498)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.util.ArrayList.forEach(ArrayList.java:1257)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:770)\r\n\tat org.testng.TestRunner.run(TestRunner.java:591)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:402)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:304)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1102)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1032)\r\n\tat org.testng.TestNG.run(TestNG.java:1000)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)\r\n"},"stage":"finished","description":"Test Case description: Returns Employee details","steps":[],"attachments":[{"name":"getEmployeeDetails Retried testcase failed and screenshot taken!","source":"59637742-0efa-467d-bc24-9ac4bf28debc-attachment","type":"text/plain"}],"parameters":[],"start":1591595899240,"stop":1591595904002,"uuid":"6c46e512-32fc-4429-b072-99a572e79bcf","historyId":"d2c00a9303e355a436e653fa1007dec","fullName":"com.enquero.api.driscolls.tests.EmployeeTest.getEmployeeDetails","labels":[{"name":"package","value":"com.enquero.api.driscolls.tests.EmployeeTest"},{"name":"testClass","value":"com.enquero.api.driscolls.tests.EmployeeTest"},{"name":"testMethod","value":"getEmployeeDetails"},{"name":"parentSuite","value":"Regression API Test Suite"},{"name":"suite","value":"Driscolls API App Test Cases"},{"name":"subSuite","value":"com.enquero.api.driscolls.tests.EmployeeTest"},{"name":"host","value":"LAPTOP-IOVO5AK9"},{"name":"thread","value":"1692@LAPTOP-IOVO5AK9.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Story name:to check if specified endpoint fetches employee details"},{"name":"severity","value":"minor"}],"links":[]}